### DISCUSSION TITLE
Show details during installation

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2760] [Ideas] [answered]
It would be helpful to be able to see what's going on during installation.  The current install is full-screen graphical and just has a percentage bar and a 1-line description of what's happening, but we can't see all the stuff it's really doing.  It would be helpful (especially if there's errors) if there were a split screen or something that showed all the verbose output that's happening real-time.

**Comment by TOoSmOotH** [comment, marked as answer]
Alt+F2 or open another terminal session
Log in as the admin user
sudo tail -f /root/sosetup.log

**Reply by innovate-support** [reply]
Perfect!  That works great
